File: backend/components/metiers/administration_publique/legacy/helpers/samples/sample_helper_legacy.test.js
// sample_helper_legacy.test.js – Test JS exemple helper legacy
const { sampleHelperLegacy } = require('./sample_helper_legacy');
test('sampleHelperLegacy fonctionne', () => {
  expect(sampleHelperLegacy({ a: 1 })).toEqual({ a: 1, helper: true });
  expect(sampleHelperLegacy()).toEqual({ helper: true, empty: true });
});

File: backend/components/metiers/administration_publique/legacy/helpers/samples/sample_helper_legacy.test.py
# sample_helper_legacy.test.py – Test Python exemple helper legacy
from .sample_helper_legacy import sample_helper_legacy

def test_sample_helper_legacy_ok():
    assert sample_helper_legacy({'a': 1}) == {'a': 1, 'helper': True}
    assert sample_helper_legacy(None) == {'helper': True, 'empty': True}
